src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -76,6 +76,7 @@ import {
     INVALID_NUMBER,
     TIME
 } from '../constants/globalConstants';
+import { leftPad } from '../utilities/generalUtilities';
```

Comment:
lint-diff complains:
```
/home/travis/build/openscope/openscope/src/assets/scripts/client/aircraft/AircraftModel.js
  79:10  error  'leftPad' is defined but never used  no-unused-vars
```
Added in 82b0803 but no longer used after e8c15b7, can be removed.